Modules/documents/Model/Document.php

```diff
@@ -49,6 +57,28 @@ public function edit($id, $id_space, $title, $id_user){
         $sql = "UPDATE dc_documents SET title=?, id_user=? WHERE id=? AND id_space=?";
         $this->runRequest($sql, array($title, $id_user, $id, $id_space));
     }
+
+    public function setVisibility($id_space, $id, $visibility, $id_ref) {
+        $sql = "UPDATE dc_documents SET visibility=?, id_ref=? WHERE id=? AND id_space=?";
+        $this->runRequest($sql, array($visibility, $id_ref, $id, $id_space));   
+    }
+
+    public function getPublicDocs($id_space){
+        $sql = "SELECT dc_documents.*, core_users.login as user FROM dc_documents INNER JOIN core_users on core_users.id=dc_documents.id_user WHERE id_space=? AND visibility=?";
```

Comment:
returns a SQL error 'id_space' is ambiguous. should be WHERE dc_documents.id_space=? AND visibility=?";

Reply:
yes